src/App.tsx

```diff
@@ -1,16 +1,64 @@
-import { Container, CssBaseline } from '@mui/material'
+import { Box, Container, CssBaseline } from '@mui/material'
 import { CustomisedAppBar } from './components/CustomisedAppBar'
 import FaceScreen from './components/FaceScreen'
+import { CardsList } from './components/CardsList'
+import { backgroundColor } from './vars'
```

Comment:
Conventional name for similar files with constants is “constants “

Reply:
fixed